Packs/CommonPlaybooks/Playbooks/playbook-Extract_Indicators_From_File_-_Generic_v2_4_5.yml

```diff
@@ -1757,10 +1759,142 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "24":
+    id: "24"
+    taskid: 4ee7917b-6813-4e61-8612-5f7116bd30eb
+    type: regular
+    task:
+      id: 4ee7917b-6813-4e61-8612-5f7116bd30eb
+      version: -1
+      name: Extract Text From QR code
+      description: Extracts the text from a QR code. The output of this script includes the output of the script "extractIndicators" run on the text extracted from the QR code.
```

Comment:
Recommend:
Extracts the text from a QR code. The output of this task includes the output of the extractIndicators script that runs on the text extracted from the QR code.